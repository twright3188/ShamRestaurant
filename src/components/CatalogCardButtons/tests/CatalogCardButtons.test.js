import React from 'react';
import renderer from 'react-test-renderer';
import { BuyButton, DetailsButton } from '../index';

describe('catalog card buttons renders correctly', () => {
    it('buy button renders correctly', () => {
        const tree = renderer.create(<BuyButton text='Buy' />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('details button renders correctly', () => {
        const tree = renderer.create(<DetailsButton text='Details' />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});