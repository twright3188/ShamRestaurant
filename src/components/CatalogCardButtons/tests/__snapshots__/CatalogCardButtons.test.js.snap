// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`catalog card buttons renders correctly buy button renders correctly 1`] = `
<View
  accessible={true}
  clickable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#399AE5",
      "borderRadius": 8,
      "justifyContent": "center",
      "maxWidth": "51%",
      "minHeight": 36,
      "minWidth": "50%",
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontFamily": "HelveticaNeue",
        "fontSize": 14,
      }
    }
  >
    Buy
  </Text>
</View>
`;

exports[`catalog card buttons renders correctly details button renders correctly 1`] = `
<View
  accessible={true}
  clickable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "borderRadius": 8,
      "justifyContent": "center",
      "maxWidth": "51%",
      "minHeight": 36,
      "minWidth": "50%",
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#399AE5",
        "fontFamily": "HelveticaNeue",
        "fontSize": 14,
      }
    }
  >
    Details
  </Text>
</View>
`;
